tests/js/common/shell/shell-index.js

```diff
@@ -266,7 +266,32 @@ function IndexSuite() {
       } catch (e2) {
         assertEqual(errors.ERROR_ARANGO_DATA_SOURCE_NOT_FOUND.code, e2.errorNum);
       }
-    }
+    },
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief create index with field names that start or end with ":"
+////////////////////////////////////////////////////////////////////////////////
+
+    testCreateInvalidField: function() {
+      const indexTypes = ["geo", "fulltext", "persistent", "hash"];
```

Comment:
"hash" is an alias type for a "persistent" index. No need to test it separately.
```suggestion
      const indexTypes = ["geo", "fulltext", "persistent"];
```